python/ql/src/semmle/python/frameworks/Django.qll

```diff
@@ -2078,7 +2099,7 @@ private module Django {
     DjangoRouteHandler() {
       exists(DjangoRouteSetup route | route.getViewArg() = djangoRouteHandlerFunctionTracker(this))
       or
-      any(DjangoViewClassDef vc).getARequestHandler() = this
+      any(DjangoViewClass vc).getARequestHandler() = this
```

Comment:
Should this (and below) use `PossibleDjangoViewClass`?

Reply:
Nope. `PossibleDjangoViewClass` might be a bit too strongly worded. Maybe it should have been called `ProbablyNotDjangoViewClass` :laughing: